src/js/actions/transform.js

```diff
@@ -209,12 +212,13 @@ define(function (require, exports) {
      *
      * @param {Document} document
      * @param {Layer} targetLayer
-     * @param {{w: number, h: number}} size
+     * @param {{w=: number, h=: number}} size
```

Comment:
Drive-by comment: I think both formats are fine. I prefer the more compact, direct representation for _small_ records, but the expanded version with many `@param` instances for bigger ones. But, either is cool with me!